src/main/java/com/amihaiemil/versioneye/Repository.java

```diff
@@ -98,4 +99,11 @@
      * @return JsonObject
      */
     JsonObject json();
+    
+    /**
+     * Delete the Repository branch from VersionEye.
```

Comment:
@SherifWaly "Deletes a the VersionEye project associated with this Github repo" -- that's what I undestand from the documentation of the endpoint which states "This Endpoint deletes a project on VersionEye!" ... However, I cannot understand why it asks for the branch name...